modules/optimization/test/tests/dirackernels/reporter_time_point_source.i

```diff
@@ -98,6 +99,7 @@
 []
 
 [Outputs]
+  exodus = true
```

Comment:
Assuming you were using this in testing, but I don't see you using this for exodiff.
```suggestion
```